config.json

```diff
@@ -67,7 +67,7 @@
       }]
     },
     "json": {
-      "transformGroup": "js",
+      "transforms": ["attribute/cti", "name/cti/pascal", "size/use-unit", "color/hex"],
```

Comment:
Are we able to make our own transform groups instead of having to repeat like this?

Reply:
Yes we can @mtblanton. I can add these in shortly.